SETUP.md

```diff
@@ -423,6 +423,14 @@ If you run into error messages about `implicit declaration of function thin_xxx`
 
 (More info [here](https://github.com/macournoyer/thin/pull/364))
 
+#### mimemagic
+
+If you run into an error message about `Could not find MIME type database in the following locations...` while installing the `mimemagic` gem, try:
+
+- `brew install shared-mime-info`
+
+(More info on mimemagic dependencies [here](https://github.com/mimemagicrb/mimemagic#dependencies).)
```

Comment:
Can we add a note that this link contains troubleshooting for other OS types, since Brew will only for work Mac?

Reply:
yes! great call, adding that now